source/common/Tools/_K3_Resource_Partitioning.rst

```diff
@@ -156,29 +240,50 @@ This tool generates different files which have the RM board config data, QoS
+|                                        |                     |                                                       |                                                            |
+|                                        |                     |                                                       | e.g. - packages/ti/boot/sbl/soc/k3/j721s2_qos_data.c       |
++----------------------------------------+---------------------+-------------------------------------------------------+------------------------------------------------------------+
+| <soc>-firewall-config.c                | U-boot              | Not yet supported                                     |                                                            |
++----------------------------------------+---------------------+-------------------------------------------------------+------------------------------------------------------------+
+| <soc>-firewall-config.c                | SBL                 | Not yet supported                                     |                                                            |
++----------------------------------------+---------------------+-------------------------------------------------------+------------------------------------------------------------+
+| sciclient_defaultBoardcfg.c            | PDK sciclient       | packages/ti/drv/sciclient/soc/V<X>/                   | e.g. - pdk/packages/ti/drv/sciclient/soc/V1/               |
+|                                        |                     |                                                       | sciclient_defaultBoardcfg.c                                |
++----------------------------------------+---------------------+-------------------------------------------------------+------------------------------------------------------------+
+| sysfw_img_cfg.h                        | k3-image-gen        | soc/<soc>/<profile>/                                  | e.g. - k3-image-gen/soc/j721e/evm/sysfw_img_cfg.h          |
++----------------------------------------+---------------------+-------------------------------------------------------+------------------------------------------------------------+
+| rm-cfg.yaml                            | U-boot              | ti-u-boot/board/ti/<soc>/                             | e.g. - ti-u-boot/board/ti/am62ax/rm-cfg.yaml               |
++----------------------------------------+---------------------+-------------------------------------------------------+------------------------------------------------------------+
+| tifs-rm-cfg.yaml                       | U-boot              | ti-u-boot/board/ti/<soc>/                             | e.g. - ti-u-boot/board/ti/am62ax/tifs-rm-cfg.yaml          |
++----------------------------------------+---------------------+-------------------------------------------------------+------------------------------------------------------------+
```

Comment:
Consider using a list-table here. Additionally please use the file role for paths.